website/snippets/_indirect-selection-definitions.md

```diff
@@ -0,0 +1,23 @@
+<VersionBlock firstVersion="1.5" >
+
+You can use the following modes to configure the behavior when performing indirect selection (with `eager` mode as the default). Test exclusion is always greedy: if ANY parent is explicitly excluded, the test will be excluded as well.
+
+The `buildable` and `cautious` modes can be useful in environments when you're only building a subset of your DAG, and you want to avoid the test failures you'd see with `eager` that are caused by unbuilt resources. You can also achieve this with [deferral](/reference/node-selection/defer).
+
+#### Eager mode
+
+By default, runs tests if any of the parent nodes are selected, regardless of whether all dependencies are met. This includes ANY tests that reference the selected nodes. Models will be built if they depend on the selected model. In this mode, any tests depending on unbuilt resources will raise an error to help you identify potential issues.
```

Comment:
The fact that it will raise an error in certain scenarios is more of a warning of a possibly undesirable side-effect rather than a feature we are trying to promote. So I'd suggest either removing "helping to identify potential issues", or finding some other way to call out this warning.

```suggestion
By default, runs tests if any of the parent nodes are selected, regardless of whether all dependencies are met. This includes ANY tests that reference the selected nodes. Models will be built if they depend on the selected model. In this mode, any tests depending on unbuilt resources will raise an error.
```